grails-app/services/au/org/ala/phyloviz/PhyloService.groovy

```diff
@@ -5,6 +5,10 @@ import grails.transaction.Transactional
 @Transactional
 class PhyloService {
 
+    void deleteVisualisation(Integer id) {
```

Comment:
looks like anyone can delete this visualisation. But I think only the creator or admin should be able to do it.